doc/man/scons.xml

```diff
@@ -6134,58 +6121,54 @@ but excluding any suffix.</para>
   </varlistentry>
 
   <varlistentry>
-  <term>dir</term>
+  <term><parameter>dir</parameter></term>
   <listitem>
 <para>The name of the directory in which the file exists.</para>
   </listitem>
   </varlistentry>
 
   <varlistentry>
-  <term>file</term>
+  <term><parameter>file</parameter></term>
   <listitem>
 <para>The file name,
 minus any directory portion.</para>
   </listitem>
   </varlistentry>
 
   <varlistentry>
-  <term>filebase</term>
+  <term><parameter>filebase</parameter></term>
   <listitem>
-<para>Just the basename of the file,
-minus any suffix
-and minus the directory.</para>
+<para>Like <parameter>file</parameter>
```

Comment:
Don't people know what basename is?

Reply:
I thought this was a simpler definition... the exisiting basename minus the suffix (which is fine) and minus the directory (which is superfluous, that's what basename is) is kind of a mixed message. The combination of names is crappy but can't change those: we have base, file and filebase, where file is basename but we don't use that term... so why introduce the term here when we already have a nearby definition (name).

Reply:
gotcha.